Formula/xtensor.rb

```diff
@@ -15,8 +15,8 @@ class Xtensor < Formula
   depends_on "cmake" => :build
 
   resource "xtl" do
-    url "https://github.com/xtensor-stack/xtl/archive/0.6.13.tar.gz"
-    sha256 "12bd628dee4a7a55db79bff5a47fb7d96b1a4dba780fbd5c81c3a15becfdd99c"
+    url "https://github.com/xtensor-stack/xtl/archive/0.6.16.tar.gz"
```

Comment:
Should be [0.6.18](https://github.com/xtensor-stack/xtl/releases) according to the [upstream README](https://github.com/xtensor-stack/xtensor/blob/9733efcfd4bd76115c822a03dd79fbea3bda6589/README.md).

```suggestion
    url "https://github.com/xtensor-stack/xtl/archive/0.6.18.tar.gz"
```